srcpkgs/ack/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'ack'
 pkgname=ack
-version=3.3.1
+version=3.4.0
 revision=1
 archs=noarch
```

Comment:
Please remove `archs=noarch` as part of the transition away from noarch.

Reply:
Done, thanx!